src/main/resources/messages/LibertyBundles.properties

```diff
@@ -34,12 +34,12 @@ action.io.openliberty.tools.intellij.actions.RefreshLibertyToolbar.description=R
 action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.text=Liberty: Run Liberty action
 action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.description=Run the selected Liberty action
 action.io.openliberty.tools.intellij.actions.AddLibertyProjectAction.text=Liberty: Add project to the tool window
-action.io.openliberty.tools.intellij.actions.AddLibertyProjectAction.description=Add the selected Liberty project to the tool window
+action.io.openliberty.tools.intellij.actions.AddLibertyProjectAction.description=Add the selected Liberty project to the tool window 
```

Comment:
please remove the extra space at the end - there is no need to have any changes on this line

Reply:
Changes : done